calendar-app/Components/CreateEvent.js

```diff
@@ -0,0 +1,222 @@
+import * as React from 'react';
+import { SafeAreaProvider } from 'react-native-safe-area-context';
+import { View, StyleSheet, TextInput, Text, Button, ScrollView } from 'react-native';
+import DateTimePickerModal from "react-native-modal-datetime-picker";
+import Footer from './Footer';
+import Header from './Header';
+import Calendar from './Calendar';
+
+function CreateEvent() {
+    const [name, setName] = React.useState('');
+    const [fromTime, setFromTime] = React.useState(new Date());
+    const [toTime, setToTime] = React.useState(new Date());
+    const [isFromPickerVisible, setFromPickerVisibility] = React.useState(false);
+    const [isToPickerVisible, setToPickerVisibility] = React.useState(false);
+    const [description, setDescription] = React.useState('');
+
+    const showFromPicker = () => {
+        setFromPickerVisibility(true);
+    };
+
+    const hideFromPicker = () => {
+        setFromPickerVisibility(false);
+    };
+
+    const handleFromConfirm = (date) => {
+        setFromTime(date);
+        hideFromPicker();
+    };
+
+    const showToPicker = () => {
+        setToPickerVisibility(true);
+    };
+
+    const hideToPicker = () => {
+        setToPickerVisibility(false);
+    };
+
+    const handleToConfirm = (date) => {
+        setToTime(date);
+        hideToPicker();
+    };
+
+    const handleCancel = () => {
+        setName('');
+        setFromTime(new Date());
+        setToTime(new Date());
+        setFromPickerVisibility(false);
+        setToPickerVisibility(false);
+        setDescription('');
+    };
+
+    const handleCreate = async () => {
+        const eventData = {
+            name,
+            fromTime,
+            toTime,
+            description,
+        };
+    
+        console.log("Creating event with data:", eventData);
+
+        try {
+            const response = await fetch('https://yourbackend.com/events', {
```

Comment:
You can leave it like this, because I will be working with this and probably changing it, but keep in mind that line API calling should be in another component.